modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/RoleResourceManager.java

```diff
@@ -526,4 +540,270 @@ public SCIMResponse updateWithPATCH(String existingId, String patchRequest, User
         return new SCIMResponse(ResponseCodeConstants.CODE_NOT_IMPLEMENTED, ResponseCodeConstants.DESC_NOT_IMPLEMENTED,
                 Collections.emptyMap());
     }
+
+    /**
+     * Check whether it is required to update all users with the patch operation.
+     *
+     * @param patchOperations Patch operation.
+     * @return Whether it is required to update all users or not with the patch operation.
+     * @throws JSONException JSONException
+     */
+    private boolean isUpdateAllUsersOperationFound(List<PatchOperation> patchOperations) throws JSONException {
+
+        for (PatchOperation patchOperation : patchOperations) {
+            String operation = patchOperation.getOperation();
+            String path = patchOperation.getPath();
+            JSONObject valuesJson = null;
+            if (StringUtils.isBlank(path)) {
+                valuesJson = (JSONObject) patchOperation.getValues();
+            }
+            if (operation.equals(SCIMConstants.OperationalConstants.REPLACE) &&
```

Comment:
follow `<Constant>.equals(<variable>)` format
```suggestion
            if (SCIMConstants.OperationalConstants.REPLACE.equals(operation) &&
```

Reply:
Fixed